paper-drawer-panel.html

```diff
@@ -757,9 +766,43 @@
 
         _moveDrawer: function(translateX) {
           this.transform(this._transformForTranslateX(translateX), this.$.drawer);
-        }
+        },
+
+        _getAutoFocusedNode: function() {
+          return Polymer.dom(this).querySelector('[autofocus]') || this.$.firstTabStop;
```

Comment:
This limits to only nodes that are actually in the light dom, and will not work for re-distribution. e.g.
```
<dom-module id="x-app">
  <template> 
    <paper-drawer-panel id="paperDrawerPanel">
      <div drawer>
        <content></content>
      </div>
      <div main>...</div>
    </paper-drawer-panel>
  </template>
</dom-module>

<x-app>
  <button>Options</button>
  <button autofocus>Settings</button>
</x-app>